xc/device.ml

```diff
@@ -942,13 +942,13 @@ module Vgpu = DaemonMgmt(struct let pid_path domid = sprintf "/local/domain/%d/v
 module PCI = struct
 
   type t = {
-    address: Xenops_interface.Pci.address;
+    address: Xcp_pci.Pci.address;
```

Comment:
This change isn't needed if you leave the `Pci` module in xenops_interface. See comments on the xcp-idl PR.